string bin_id
string object_id
---
bool success
string message
sensor_msgs/Image mask